examples/drydock.conf

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
```

Comment:
This entire file can be auto-generated. I believe Alex recently did this in Armada using oslo-config-generator. That job can then be added to tox.ini if you have one in this repo.